<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updatehealthform</class>
 <widget class="QWidget" name="updatehealthform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="header">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>65</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>734</width>
       <height>65</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget #header {
	background-color: rgb(255, 255, 255);
	background-image:url(:/images/update_header.png);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">QScrollBar:horizontal {
	background-color: white;
    height: 14px;
    margin: 0px 20px 0px 20px;
}

QScrollBar::handle:horizontal {
	background-color: rgb(195, 195, 195);
    min-width: 15px;
}

QScrollBar::add-line:horizontal {
	border-image: url(:/images/scroll_right.png);
    width: 16px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal {
	border-image: url(:/images/scroll_left.png);
    width: 16px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}

QScrollBar:vertical {
	width:14px;
	background-color: white;
	margin: 20px 0px 20px 0px;
}

QScrollBar::handle:vertical {
	background-color: rgb(195, 195, 195);
    min-height: 15px;
}

QScrollBar::add-line:vertical {
	border-image: url(:/images/scroll_down.png);
    height: 16px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical {
	border-image: url(:/images/scroll_up.png);
    height: 16px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QFrame {
	background-color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:5px;
border: 1px solid rgb(170, 170, 170);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(254, 254, 254), stop: 1 rgb(252, 252, 252));
}

QLineEdit:hover {
border-radius:5px;
border: 1px solid rgb(23, 120, 173);
background-color: rgb(255, 255, 255);
}

QLineEdit:pressed {
border-radius:5px;
border: 1px solid #777777;
}

QComboBox {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:5px;
border: 1px solid rgb(170, 170, 170);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(254, 254, 254), stop: 1 rgb(248, 248, 248));
}

QComboBox:hover {
color: rgb(255, 255, 255);
border-radius:5px;
border: 1px solid rgb(23, 120, 173);
background-color: ;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(161, 215, 255), stop: 1 #299ED6);
}

QComboBox:pressed {
border-radius:5px;
border: 1px solid #777777;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px;
 
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
	image: url(:/images/cmbbox_arrow.png);
}

QDateEdit {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:5px;
border: 1px solid rgb(170, 170, 170);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(254, 254, 254), stop: 1 rgb(252, 252, 252));
}

QDateEdit:hover {
border-radius:5px;
border: 1px solid rgb(23, 120, 173);
background-color: rgb(255, 255, 255);
}

QDateEdit:pressed {
border-radius:5px;
border: 1px solid #777777;
}

QDateEdit::down-button {
	subcontrol-origin: padding;
	subcontrol-position: bottom right;
	width: 25px;
 
	border-top-right-radius: 1px;
	border-bottom-right-radius: 1px;
}

QDateEdit::down-button:hover {
	background-color: rgb(227, 227, 227);
}


QDateEdit::down-arrow {
	image: url(:/images/scroll_down.png);
}

QDateEdit::up-button {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px;
 
	border-top-right-radius: 1px;
	border-bottom-right-radius: 1px;
}

QDateEdit::up-button:hover {
	background-color: rgb(227, 227, 227);
}

QDateEdit::up-arrow {
	image: url(:/images/scroll_up.png);
}

QPlainTextEdit {
color: rgb(50, 50, 50);
padding-left:7px;
border-radius:5px;
border: 1px solid rgb(170, 170, 170);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 rgb(254, 254, 254), stop: 1 rgb(252, 252, 252));
}

QPlainTextEdit:hover {
border-radius:5px;
border: 1px solid rgb(23, 120, 173);
background-color: rgb(255, 255, 255);
}

QPlainTextEdit:pressed {
border-radius:5px;
border: 1px solid #777777;
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="horizontalSpacing">
          <number>9</number>
         </property>
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Inmate ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txtID">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Date:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="deDate">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Urgency:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cmbUrgency">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>High</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Low</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Inmate health condition:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="txtCondition">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>27</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Prescriptions/Remarks:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="txtRemarks">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>180</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QFrame" name="frame_7">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>label_12</zorder>
        <zorder>txtID</zorder>
        <zorder>label_7</zorder>
        <zorder>label_11</zorder>
        <zorder>label_8</zorder>
        <zorder>frame_7</zorder>
        <zorder>label_16</zorder>
        <zorder>txtCondition</zorder>
        <zorder>deDate</zorder>
        <zorder>cmbUrgency</zorder>
        <zorder>txtRemarks</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(160, 160, 160);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>48</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 5px;
border: 1px solid rgb(150, 150, 150);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #ccc);
}

QPushButton:hover {
border-style: inset;
background-color: #FFFFFF;
border: 2px solid #299ed6;
}

QPushButton:pressed {
border: 1px solid rgb(140, 140, 140);
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #999, stop: 1 #ddd);
}

#frame_5 {
border: none;
background-color: rgb(205, 205, 205);
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="styleSheet">
         <string notr="true">border: none;
background-color:0;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="pms_resources.qrc">
          <normaloff>:/images/save_data.png</normaloff>:/images/save_data.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>27</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>27</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="pms_resources.qrc">
          <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pms_resources.qrc"/>
 </resources>
 <connections/>
</ui>
